spec/helpers/session.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../factories/user.rb'
+
+
+module SessionHelpers
+
+  def sign_in(user)
+    visit '/sessions/new'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)